### DISCUSSION TITLE
How to add navigation bar along right side of screen

### DISCUSSION THREAD

**Discussion by p-jones** [docmost/docmost#89] [Help / Questions]
I wonder if I am missing a basic "how to" document since I am just trying to create a clickable navigation bar (outline) along the right side of the screen.

I had assumed using H1 headings would accomplish this, but it doesn't appear to.

Any help much appreciated!

**Comment by Nitrox912** [comment]
Is this feature planned for the near future?

**Reply by Philipinho** [reply]
@p-jones @Nitrox912 do you mean table of contents of headings?

**Reply by Nitrox912** [reply]
@Philipinho exactly 👍

**Comment by LQ753799168** [comment]
Hopefully it will be similar to markdown's outline functionality
